changelogs/fragments/816-only-invocate-feature-when-variable-is-set.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - proxmox lxc - only invocate feature flag, when variable is set, 
+  - proxmox lxc - an empty feature variable breaks proxmox pct list 
+  - proxmox lxc - feature invocation forces to use root@pam for module execution  
```

Comment:
```suggestion
  - proxmox lxc - only add the features flag when module parameter ``features`` is set. Before an empty string was send to proxmox in case the parameter was not used, which required to use ``root@pam`` for module execution (https://github.com/ansible-collections/community.general/pull/1763).
```